docs/workflow/building/coreclr/README.md

```diff
@@ -1,56 +1,100 @@
+Or alternatively:
+
+```bash
+./build.sh --subset clr+libs --librariesConfiguration Release --runtimeConfiguration Debug
 ```
-./build.cmd -subset clr -ninja
+
+For more information about all the different options available, supply the argument `-help|-h` when invoking the build script. On Unix-like systems, non-abbreviated arguments can be passed in with a single `-` or double hyphen `--`.
+
+### Build Drivers
+
+If you want to use _Ninja_ to drive the native build instead of _Make_ on non-Windows platforms, you can pass the `-ninja` flag to the build script as follows:
+
+```bash
+./build.sh --subset clr --ninja
 ```
 
-If you want to use Visual Studio's MSBuild to drive the native build on Windows, you can pass the `-msbuild` flag to the build script similarly to the `-ninja` flag.
+If you want to use Visual Studio's _MSBuild_ to drive the native build on Windows, you can pass the `-msbuild` flag to the build script similarly to the `-ninja` flag.
 
-We recommend using Ninja for building the project on Windows since it more efficiently uses the build machine's resources for the native runtime build in comparison to Visual Studio's MSBuild.
+We recommend using _Ninja_ for building the project on Windows since it more efficiently uses the build machine's resources for the native runtime build in comparison to Visual Studio's _MSBuild_.
+
+### Extra Flags
 
 To pass extra compiler/linker flags to the coreclr build, set the environment variables `EXTRA_CFLAGS`, `EXTRA_CXXFLAGS` and `EXTRA_LDFLAGS` as needed. Don't set `CFLAGS`/`CXXFLAGS`/`LDFLAGS` directly as that might lead to configure-time tests failing.
 
-This will produce outputs as follows:
+### Build Results Layout
+
+Once the build has concluded, it will have produced its output artifacts in the following structure:
 
-- Product binaries will be dropped in `artifacts\bin\coreclr\<OS>.<arch>.<flavor>` folder.
-- A NuGet package, Microsoft.Dotnet.CoreCLR, will be created under `artifacts\bin\coreclr\<OS>.<arch>.<flavor>\.nuget` folder.
-- Test binaries will be dropped under `artifacts\tests\coreclr\<OS>.<arch>.<flavor>` folder. However, the root build script will not build the tests.
+* Product binaries will be dropped in `artifacts\bin\coreclr\<OS>.<arch>.<configuration>` folder.
+* A NuGet package, _Microsoft.Dotnet.CoreCLR_, will be created under `artifacts\bin\coreclr\<OS>.<arch>.<configuration>\.nuget` folder.
+* Test binaries (if built) will be dropped under `artifacts\tests\coreclr\<OS>.<arch>.<configuration>` folder. However, remember the root build script will not build the tests. The instructions for working with tests (building and running) are [in the testing doc](/docs/workflow/testing/coreclr/testing.md).
+* The build places logs in `artifacts\log` and these are useful when the build fails.
+* The build places all of its intermediate output in the `artifacts\obj\coreclr` directory, so if you remove that directory you can force a full rebuild.
```

Comment:
It might be worth mentioning that running <code>src/tests/build.sh/cmd</code> with the <code>rebuild</code> option does just this - deletes the <code>artifacts\obj\coreclr</code> and <code>artifacts\tests\coreclr</code> subtrees upfront, please see here:

https://github.com/dotnet/runtime/blob/415a41770bdf8efd4c3217e2c281233ee5cd03ea/src/tests/build.cmd#L144
https://github.com/dotnet/runtime/blob/415a41770bdf8efd4c3217e2c281233ee5cd03ea/src/tests/build.sh#L14